.stylelintrc

```diff
@@ -1,13 +1,13 @@
 {
   "enabled": true,
-  "extends": [
-    "stylelint-config-recommended-scss"
-  ],
-  "plugins": [
-    "stylelint-scss"
-  ],
+  "extends": ["stylelint-config-recommended-scss"],
+  "plugins": ["stylelint-scss"],
   "rules": {
     "block-no-empty": null,
-    "declaration-block-single-line-max-declarations": 1
+    "declaration-block-single-line-max-declarations": 1,
+    "property-blacklist": null,
+    "unit-no-unknown": null,
+    "at-rule-no-unknown": null,
+    "property-no-unknown": null
```

Comment:
On this, is there no way to allow the offending things rather than disabling the check? I was hoping that upgrading `stylelint` might mean that it recognises `dvh` for example, so we could keep the rule in place 🤔 If not we can always add a list of ignored units as per https://stylelint.io/user-guide/rules/unit-no-unknown/